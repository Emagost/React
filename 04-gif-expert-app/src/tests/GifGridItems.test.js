import React from 'react';
import { shallow } from 'enzyme';
import GifGridItem from '../components/GifGridItem';


describe('Test GifGridItme', () => {
    test('should show corretly the component', () => {
        const wrapper = shallow(<GifGridItem />);
        expect(wrapper).toMatchSnapshot();
    })
    
})
