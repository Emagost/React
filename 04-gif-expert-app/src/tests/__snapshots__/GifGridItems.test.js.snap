// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test GifGridItme should show corretly the component 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <img />
  <p>
     
     
  </p>
</div>
`;
